doc/how_to_use.md

```diff
@@ -0,0 +1,131 @@
+# How to use Sōzu
+
```

Comment:
do not start with the configuration file. Start by explaining roughly how it will run, what are the master and workers, the differences between application, frontend and backend, how it can get its configuration. There will be some overlap with the design doc, but it's fine